web/scripts/retrieve_data.py

```diff
@@ -52,7 +52,8 @@
     "江行智能": "Jiangxing Intelligence",
     "智易科技": "Zhiyi Tech",
     "创新奇智": "AInnovation",
-    "captricity": "Vidado"
+    "captricity": "Vidado",
+    "Alphabet": "Alphabet (including Google)"
```

Comment:
Okay, yes, it looks like we've decided to roll up Google?

Reply:
Yep, see https://github.com/georgetown-cset/parat/issues/284